videointelligence/snippets/object_tracking.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package snippets contains speech examples.
+package snippets
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+
+	// [START imports]
+	"context"
+
+	"github.com/golang/protobuf/ptypes"
+
+	video "cloud.google.com/go/videointelligence/apiv1"
+	videopb "google.golang.org/genproto/googleapis/cloud/videointelligence/v1"
+	// [END imports]
+)
+
+// [START video_object_tracking]
+
+func objectTracking(w io.Writer, filename string) error {
+	ctx := context.Background()
+
+	// Creates a client.
+	client, err := video.NewClient(ctx)
+	if err != nil {
+		log.Fatalf("Failed to create client: %v", err)
+	}
+
+	// filename = "../resources/cat.mp4"
+	fileBytes, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return err
+	}
+
+	op, err := client.AnnotateVideo(ctx, &videopb.AnnotateVideoRequest{
+		InputContent: fileBytes,
+		Features: []videopb.Feature{
+			videopb.Feature_OBJECT_TRACKING,
+		},
+	})
+	if err != nil {
+		log.Fatalf("Failed to start annotation job: %v", err)
+	}
+
+	resp, err := op.Wait(ctx)
+	if err != nil {
+		log.Fatalf("Failed to annotate: %v", err)
+	}
+
+	// Only one video was processed, so get the first result.
+	result := resp.GetAnnotationResults()[0]
+
+	for _, annotation := range result.ObjectAnnotations {
+		fmt.Fprintf(w, "Description: %s\n", annotation.Entity.GetDescription())
+		if len(annotation.Entity.EntityId) > 0 {
```

Comment:
Print this every time and use `%q` to get automatic quotes around it?

Reply:
Done